### ISSUE TITLE
Non-standard keyboard commands required to navigate and edit keyboards shortcuts in Settings dialogue using screen readers

### ISSUE THREAD

**Issue by togregg** [edrlab/thorium-reader#2601]
### Impact: Minor
[WCAG 2.1 SC 1.3.1 - Info and Relationships (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships) 
[WCAG 2.1 SC 1.3.2 - Meaningful Sequence (Level A)](https://www.w3.org/WAI/WCAG21/Understanding/meaningful-sequence)

### Current Behavior:
From the Settings dialogue, once the Keyboard shortcuts tab is selected, and you Tab over to the options for Keyboard Shortcut, you cannot use standard screen reader keyboard commands to navigate through the edit buttons for Keyboard shortcuts. In order to get to the edit button for each keyboard shortcut you must press Tab. This means you can't use the standard screen reader commands for navigation and end up missing the heading and note about QWERTY layout above the keyboard shortcuts.

### Example:
Using VoiceOver while in the settings dialogue, navigate to the Keyboard Shortcuts tab, activate it and then tab over to the options for Keyboard shortcuts. Using VO+ Right arrow to navigate you go past the Keyboards shortcuts heading, menu, Close window button, text saying “Shortcuts were defined from QWERTY keyboard layout.”. Pressing VO + Right arrow again VoiceOver announces end of keyboard shortcuts tab panel. You can’t go any further with standard navigation command. To access the edit buttons, you must tab to each one.

### Expected Behavior:
From the Settings dialogue, once the Keyboard shortcuts tab is selected, and you Tab over to the options for Keyboard Shortcut, you can use standard screen reader keyboard commands to navigate through the edit buttons for Keyboard shortcuts.

#### Windows
- Windows 11 Pro Version 22H2 OS build 22621.3447
- JAWS 2024.2409.2
- NVDA 2024.1 (2024.1.0.31547)
- Thorium v3.0.0

#### macOS
- Sonoma 14.6.1 (23G93)
- Voice Over for Sonoma
- Thorium 3.0.0

### Steps to recreate:

1. Start your screen reader of choice.
2. Launch Thorium 3.0.0.
3. Using the standard navigation keyboard commands for your screen reader, navigate to and activate the Settings button.
   - JAWS/NVDA – Down arrow 
   - VoiceOver – Ctrl + Option + Right Arrow
4. From the dialogue that appears, navigate up from the Close window button until you get to Keyboards shortcuts tab.
5. Press Tab to go to the contents for Keyboard Shortcuts.
6. Use the standard navigation keyboard commands for your screen reader, go down to the text “Shortcuts were defined from QWERTY keyboard layout”.
   - JAWS/NVDA – Down arrow
   - VoiceOver – Ctrl + Option + Right Arrow
7. Try using the standard navigation command again…nothing should happen (outside of silence or a ding, depending on your screen reader).
8. Press Tab and you will be on the first edit button for a keyboard shortcut.

### Video(s)/Screenshots:

- [Thorium 3 Issue - In the Settings dialogue accessing the edit buttons for keyboard shortcuts does not behave as expected with screen readers (VoiceOver)](https://iu.mediaspace.kaltura.com/media/t/1_a1yqdzxi)
- [Thorium 3 Issue - In the Settings dialogue accessing the edit buttons for keyboard shortcuts does not behave as expected with screen readers (JAWS/NVDA)](https://iu.mediaspace.kaltura.com/media/t/1_bsc4nrby)

### Recommendations:

- Expose the keyboard shortcuts and their edits buttons so standard keyboard commands used by screen readers can access them.
- Expose all of the keyboard shortcuts so a screen reader can use options like elements list or forms field to see all of the keyboard shortcut edit buttons.
